### ISSUE TITLE
M > Translate and review the develop video subtitle

### ISSUE THREAD

**Issue by zsluedem** [rchain/bounties#347]
Recently I watched the develop videos from the [Rchain Youtube Channel](https://www.youtube.com/watch?v=Fs38s7nI3Uk). These videos really helped me understand the Rchain ideas.And for those who doesn't speak English would have difficulties to understand these videos.I think translation of these videos would be great for people to know Rchain better.

I found a really [*USEFUL TOOL*](https://github.com/1c7/Youtube-Auto-Subtitle-Download) from github.And you can download the subtitle from the Youtube(autogenerated or not). I know the Youtube autogenerated subtitle is *NOT PERFECT*.But this tool really reduces our subtitle translating job.Now we only have to do the reviewing.

I have uploaded all these autogenerated [subtitle on the Google drive](https://drive.google.com/drive/u/0/folders/1IswQyoid2ZUQywgMkVY9HXqeVYYM0vIh).

I have also count the amount of words on each subtitle.So we can setup the budget properly(But the autogenerated subtitle hasn't been reviewed,so the word count would be a little different.).

Recommend buget for each subtitle(the buget is based on @lapin7 on #251 [comment](https://github.com/rchain/Members/issues/251#issuecomment-361984792), $0.2 for 1 English word regardless of numbers or something else):

- [x] [Architecture & Rholang Overview](https://www.youtube.com/watch?v=Fs38s7nI3Uk&list=PLf2bbiic5ZjCPzin3gCSMGiBtbT8UO5o2): [English subtitle(finish reviewed)](https://docs.google.com/document/d/1vZuNiZ0DH-QBuRwkZpHFAxFSXgROk8_4FOBIWLiGnf4/edit) total words: 9447
 
- [x] [Blockchain Use Case: Supply Chain](https://www.youtube.com/watch?v=vFM61JXabsY&index=2&list=PLf2bbiic5ZjCPzin3gCSMGiBtbT8UO5o2): [English subtitle(finish reviewed)](https://docs.google.com/document/d/1hiJ6QShQOoOBNd-5812Q43RhxtzGVQ-LluZbHP1PeQs/edit) total words: 3946 
- [x] [Block formats, Types, and Messaging](https://www.youtube.com/watch?v=7jMgsdSHsy0&index=3&list=PLf2bbiic5ZjCPzin3gCSMGiBtbT8UO5o2): [English subtitle(finish reviewed)](https://docs.google.com/document/d/1ZUo58WyG0Vk-fUsG0DBwgHXghFU5YGCU3SmCtST55Xw/edit) total words: 3397 
- [x] [Compiler Design, Cryptoeconomics](https://www.youtube.com/watch?v=sMEZmm3uzH8&list=PLf2bbiic5ZjCPzin3gCSMGiBtbT8UO5o2&index=4): [English subtitle(not reviewed)](https://docs.google.com/document/d/1fMTWXyymsbNX7tmh3471iTIG4LfHFFCWqCp_UknE110/edit?usp=drive_web&ouid=115573881377148283245) total words: 9712 
- [ ] [Rholang 101: Syntax and Semantics](https://www.youtube.com/watch?v=AUoRN9PxOcw&list=PLf2bbiic5ZjCPzin3gCSMGiBtbT8UO5o2&index=5): [English subtitle(not reviewed)](https://docs.google.com/document/d/1ukL2KGLJBvlw7ihmS6jMiPuzGN_AK53zN8rUlic0MkU/edit) total words: 7924 
- [x] [Blockchain Business Requirements](https://www.youtube.com/watch?v=OcTM8rKEjlk&list=PLf2bbiic5ZjCPzin3gCSMGiBtbT8UO5o2&index=6): [English subtitle(finish reviewed)](https://docs.google.com/document/d/1eBlzV9wi9n2SguBx6qaMiq2YFnXeiABayfhT7QlbY1k/edit?usp=drive_web&ouid=115573881377148283245) total words: 3166 
- [x] [Rholang Smart Contracts Lab](https://www.youtube.com/watch?v=0wi0txVf0Po&list=PLf2bbiic5ZjCPzin3gCSMGiBtbT8UO5o2&index=7): [English subtitle(finish  reviewed)](https://docs.google.com/document/d/1NaYG0MnanalQ7DTMIthDYM2Mj9alMghhLx_ew-XL_Kk/edit) total words: 3017 
- [ ] [Casper Proof-of-Stake on RChain](https://www.youtube.com/watch?v=qJFCtmlCu-0&list=PLf2bbiic5ZjCPzin3gCSMGiBtbT8UO5o2&index=8): [English subtitle(not reviewed)](https://docs.google.com/document/d/1x3D4Bph8MknZoAhy3lV7BiS3PaT-0Sq7N3tJHqdHRPo/edit?usp=drive_web&ouid=115573881377148283245) total words: 3050 
- [ ] [Casper Proof-of-Stake on RChain Part 2](https://www.youtube.com/watch?v=GGdri2iabOk&index=9&list=PLf2bbiic5ZjCPzin3gCSMGiBtbT8UO5o2): [English subtitle(not reviewed)](https://docs.google.com/document/d/1wATsVIOm6GhzSCftp3azBMjrXlMecwmHfyaFhSophoc/edit) total words: 1554 
- [ ] [RChain Architecture & Implementation detailed](https://www.youtube.com/watch?v=bwH66UMFp-Y&list=PLf2bbiic5ZjCPzin3gCSMGiBtbT8UO5o2&index=10): [English subtitle(not reviewed)](https://docs.google.com/document/d/1ZngNoLhihen2Aqwv6yU7rPpbJnDi8LV8HKhEt6zFBwQ/edit?usp=drive_web&ouid=115573881377148283245) total words: 5110 
- [ ] [Go-To-Market Strategy](https://www.youtube.com/watch?v=A0wZ0Y4zlFY&index=11&list=PLf2bbiic5ZjCPzin3gCSMGiBtbT8UO5o2): [English subtitle(not reviewed)](https://docs.google.com/document/d/1NBE9LLxZvIkst824dubG7lAlh2of6lQw2o_P1BXXq-Q/edit) total words: 2969
- [x] [Roscala Interpretation for VM](https://www.youtube.com/watch?v=CkIOWSG_PMI&index=12&list=PLf2bbiic5ZjCPzin3gCSMGiBtbT8UO5o2): [English subtitle(not reviewed)](https://docs.google.com/document/d/1CzE8PE4Q3NScj4OKgn4rJcoWIvE-OXNIF9MmeSnen8Y/edit) total words: 1015
- [ ] [LADL Algorithm and Type Theory](https://www.youtube.com/watch?v=rZawYtTdObo&list=PLf2bbiic5ZjCPzin3gCSMGiBtbT8UO5o2&index=13): [English subtitle(not reviewed)](https://docs.google.com/document/d/1hnXAJH3dwODPYo0o8YQ1zr9iYi08NPtYAmsH4ywqE4A/edit) total words: 14305
- [ ] [Codebase Documentation](https://www.youtube.com/watch?v=5FHomk__7-k&list=PLf2bbiic5ZjCPzin3gCSMGiBtbT8UO5o2&index=14): [English subtitle(not reviewed)](https://docs.google.com/document/d/1RrQMTWZAFEmHmO8UveoAv4T74hWlyM0ZOD2N6EIgjOk/edit) total words : 1851
- [x] [Release Management](https://www.youtube.com/watch?v=tjLGuvuc0dI&list=PLf2bbiic5ZjCPzin3gCSMGiBtbT8UO5o2&index=15): [English subtitle(not reviewed)](https://docs.google.com/document/d/12dylA413NqLFbn3ksh1BtF4AxDhHgmrLnwLLXuudoiQ/edit) total words: 1494
- [x] [Rholang Smart Contracts Lab part 2](https://www.youtube.com/watch?v=2DIevavKPwc): [English subtitle(finish reviewed)](https://docs.google.com/document/d/1-PBL9YSRKJYXAtCf2lGe8UFzUjg207fd4GtqMMnX1lU/edit) total words: 3017

Based on my translation experience on the `Layrence and Nash interview` , I think the most efficient way to do the reviewing is tranlating.

Feel free to open an issue for translating one of the subtitle above.Remember to review the English subtitle.A Native speaker is required.Hope everyone can reference the advice from #302.Keep the translation high quality.

*TIME*:huge amount of work, 2 months

*BUDGET*: $500 (only for this issue)

**Comment by Whisker17** [comment]
cool

**Comment by Ojimadu** [comment]
I think the most important thing first is the review of the transcript as it usually contains lots of errors that on its own can affect the quality of translation work done.

**Comment by Viraculous** [comment]
I second @Ojimadu,  and for better precision in translation, technical terms could be extracted  and reviewed for precise word replacements in translation by native experts with basic knowledge programming language and blockchains e.g I heard Nash Foster mentioned "Scope of extrusion" in the interview with Brad.  That's a technical phenomenon. I believe much more will be encountered

**Comment by han014** [comment]
cool!

**Comment by han014** [comment]
I will translate the first video recent days and then submit.

**Comment by Whisker17** [comment]
I'll review the first one "Architecture & Rholang Overview" and translate with @han014

**Comment by han014** [comment]
@lapin7 I need the access to Google Cloud Drive and review reward. Thanks.

**Comment by Ojimadu** [comment]
@han014 Try messaging @BelovedAquila as well.

**Comment by han014** [comment]
@BelovedAquila I need the access to Google Cloud Drive and review reward. Thanks.

**Comment by hcl3** [comment]
cool!

**Comment by zsluedem** [comment]
Anyone who try to review the subtitle please self-assign or make a comment.

**Comment by han014** [comment]
I will translate the video Blockchain Business Requirements recent days and then submit.

**Comment by rjl493456442** [comment]
@zsluedem Hey, i create an issue to translate one of the video subtitle to chinese. I'll help you to review the original subtitle and upload the revision to the given drive.

**Comment by AbnerZheng** [comment]
@zsluedem I will create a new issue to translating ' LADL Algorithm and Type Theory', I am quite interested in this theory, thanks for you effort.

**Comment by yyh1102** [comment]
@zsluedem I'll create an  issue to translate ' Rholang Smart Contracts Lab'.

**Comment by Whisker17** [comment]
Glad to see some many people join in this issue,I'll upload the " Blockchain Business Requirements" reviewed subtitle and the its Chinese version.
Also,I and @han014 will review the "Compiler Design, Cryptoeconomics" and translate it into Chinese recent days.

**Comment by zsluedem** [comment]
@han014 @rjl493456442 @AbnerZheng @yyh1102 Feel free to open a seperate issue. And I really suggest you can collaborate with each other for reviewing or something.That would be helpful for budget voting.

**Comment by yyh1102** [comment]
@zsluedem OK. I can help to review the work of @rjl493456442 ~

**Comment by yuhuanmocmoc** [comment]
@zsluedem Hey, i create an issue #437 to translate the “Go-To-Market Strategy” to chinese,thanks for you effort.

**Comment by linsheng9731** [comment]
@zsluedem Great work! I will create a issue to translating ' RChain Architecture & Implementation detailed: English subtitle', I am interested in it.

**Comment by Whisker17** [comment]
@linsheng9731 @yyh1102  @yuhuanmocmoc @rjl493456442 @AbnerZheng  I've create this [folder](https://drive.google.com/drive/u/0/folders/1BvOKt0ThjJeV2hxJ-Y_feJ-n2KZkEHkC) to upload reviewed subtitle,you can create the folder like me to make it clearer,thanks.

**Comment by yyh1102** [comment]
@zsluedem I found that the total words of 'Rholang Smart Contracts Lab part 1' is not correct. I'll declare the correct amount of words in #432 ~

**Comment by zsluedem** [comment]
@yyh1102 how many words do you count that? After you finish reviewing the transtitle, the word count would change. It is normal. It is not 100% accurate , I mentioned above.

**Comment by yyh1102** [comment]
@zsluedem OK, i just give a reminder~
 The number of words of 'Rholang Smart Contracts Lab  part 1' before reviewing is around 9000.

**Comment by watermelonZJ** [comment]
@zsluedem Hey, i'd love to take up the `codebase document` video subtitle's translation.

**Comment by zsluedem** [comment]
@watermelonZJ feel free to open an issue. Remember to find some other collaborator to do together.

**Comment by zsluedem** [comment]
@yyh1102 [here](https://drive.google.com/drive/u/0/folders/1BvOKt0ThjJeV2hxJ-Y_feJ-n2KZkEHkC)

**Comment by yyh1102** [comment]
Have complete translation work of 'Rholang Smart Contracts Lab' with part1 and part2' in #432 ~ Have upload to [here](https://drive.google.com/drive/u/0/folders/1Ay8Fdmh9uvjnvDuhBW3gD2Y588BBRV-D).

**Comment by lapin7** [comment]
Please split this issue like:
- each single document or video
- each language
And create separate issues with links to this mother issue

**Comment by han014** [comment]
I'd like to review the videos ''Roscala Interpretation for VM: English subtitle'' and ''Release Management: English subtitle'', and then translate them into Chinese with @Whisker17 in recent days.

**Comment by watermelonZJ** [comment]
@zsluedem Hey will, i have finished the translation. Please create the directory for `Codebase Documentation` and i'll upload the relative files later. Thanks for your cooperation :)

**Comment by zsluedem** [comment]
@watermelonZJ [here](https://drive.google.com/drive/u/0/folders/10VxItrP5iQA1GKquLfgjKWMQ8mrqaeDM)

**Comment by watermelonZJ** [comment]
@zsluedem All relative files have been uploaded! Please take a look and update the video status if available, thanks.

**Comment by zsluedem** [comment]
Who is responsible with the Blockchain Use Case: Supply Chain and  Block formats, Types, and Messaging? If there is nobody doing it , I would assign somebody.

I don't know who click `done` with these two.

**Comment by rjl493456442** [comment]
@zsluedem Please make a room for me to upload the reviewed subtitle and translation, thanks.

**Comment by ian-bloom** [comment]
I do not think that videos or video transcripts are good targets for translation. To make content more accessible, videos should first be summarized into blog posts, and then **only the summaries should be translated.** This makes the content far easier to access and can be consumed in less time. Blog summaries of videos can contain screenshots and links to particular times in the video so that the readers know exactly where to go when they need more detail than provided by the summary.

Videos from the November 2017 developers retreat have lost much of their relevance. The project has advanced significantly since November, and the next Dev-Retreat is less than 3 weeks away, in Boulder Colorado. Those sessions will be recorded and hosted on YouTube as well.

**Comment by zsluedem** [comment]
@rjl493456442 which one you are working on?

**Comment by rjl493456442** [comment]
@zsluedem  I'm working on the` Rholang 101: Syntax and Semantics`

**Comment by zsluedem** [comment]
@ian-bloom actually you make a point. When I heard of the next Dev-Retreat, I struggled a little if I should close this issue. But some of the videos has been translated to Chinese. I have baked the subtitle to the video and uploaded them to the Chinese website these days.
I want to see if the clicks and views of these videos would be much more than the original English videos. The outcome effect of the videos depend on necessary the translation of these videos .
If those videos are still no-one-view videos, then the translations of those video are not useful.
Then I think we should stop those translations.
